fog/test-client/src/test_client.rs

```diff
@@ -669,33 +684,50 @@ impl TestClient {
     }
 
     /// Conduct an atomic swap transfer between two clients
-    /// Returns the transaction and the block count of the node it was submitted
-    /// to.
     ///
-    /// The target client builds a signed contingent input, then the source
-    /// client incorporates this into a Tx and submits it.
+    /// The source client builds an SCI, and the target client builds the swap
+    /// transaction. This only builds and submits the transactions, it does not
+    /// confirm it.
```

Comment:
i decided its more logical for the source client to be the originator and build the SCI, and the target client should be the counterparty that fills it

Reply:
Maybe we should use that language then, if that's how we're wording it? If we call them sci originator, and sci counterparty would that add clarity?

Reply:
sure, but at a higher level in the test, the language source-client and target-client is used. but in this function i guess we could do that

Reply:
Let's make an issue to track this in a subsequent PR, I plan to make some follow-on changes for the sci interface anyways.

Reply:
I changed the code comment, but not the variable names